UserService/UserService.Handlers/GetNewsTickerHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MediatR;
+using System.Threading;
+using System.Threading.Tasks;
+using UserService.Core.Interfaces.Repositories;
+using HelpMyStreet.Contracts;
+
+namespace UserService.Handlers
+{
+    public class GetNewsTickerHandler : IRequestHandler<NewsTickerRequest, NewsTickerResponse>
+    {
+        private readonly IRepository _repository;
+
+        public GetNewsTickerHandler(IRepository repository)
+        {
+            _repository = repository;
+        }
+
+        public async Task<NewsTickerResponse> Handle(NewsTickerRequest request, CancellationToken cancellationToken)
+        {
+
+            return new NewsTickerResponse()
+            {
+                Messages = await _repository.GetNewsTickerMessages(request.GroupId, cancellationToken)
```

Comment:
I think the repository should have 1-3 methods to supply the numbers, but the service layer should do the work to formulate the messages, decide which ones to display, etc.